app/presenters/place_presenter.rb

```diff
@@ -0,0 +1,29 @@
+class PlacePresenter < ContentItemPresenter
+  PASS_THROUGH_DETAILS_KEYS = %i(
+    introduction
+    more_information
+    need_to_know
+    place_type
+  ).freeze
+
+  PASS_THROUGH_DETAILS_KEYS.each do |key|
+    define_method key do
+      details[key.to_s] if details
+    end
+  end
+
+  def initialize(content_item, places = nil)
```

Comment:
Could places default to an empty array instead? Might save a couple of lines of code...